docs: add cloud info to readme (#82)
reearth/reearth: README.md

For examples and more, please refer to the [Documentation](https://docs.reearth.io/)

## Cloud

> We also maintain an cloud service [here](https://reearth.io/), which can help you start your Re:Earth journey more easily

Reviewer: ```suggestion
We also maintain a cloud service [here](https://reearth.io/), which can help you start your Re:Earth journey quicker.
```

Author: Thanks 🙏
